### PR TITLE
Proj0025: Migrate from ruleset file to .editorconfig file

### PR DESCRIPTION
**Pull request by Corniel**
Documentation for https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/pull/179

### CODE REVIEW

File: rules/Proj0025.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Proj0025: Migrate from ruleset file to .editorconfig file
+Microsoft depricated the of `.ruleset`'s in favour of .editorconfig files.
```
Comment (Gemberkoekje) [inline rules/Proj0025.md:2]: "depricated the of '.rulesets'..."
Should be
"depricated the use of '.rulesets'..."
or just "depricated '.rulesets'..."

**Review by Gemberkoekje — APPROVED** [review]
Small fix on the text, good other than that.

File: rules/Proj0025.md (hunk 1 above)
Comment (LauraXiulan) [reply rules/Proj0025.md:2]: deprecated ;)

**Review by LauraXiulan — APPROVED** [review]
See above comment

**Pull request merged**